src/DataTable.scss

```diff
@@ -0,0 +1,37 @@
+.chw-table {
```

Comment:
Look much better with a separate file.

I agree with what you mentioned about the css: we need to better organize the css files. One related issue is that some css styles, such as `.chw-button-sm`, are used in diverse js files, so it looks we need to put commonly-used styles in a separate css file with better naming.

Reply:
I have added this issue https://github.com/hms-dbmi/cistrome-higlass-wrapper/issues/164

Reply:
Thanks!